independent-projects/tools/platform-descriptor-api/src/main/java/io/quarkus/platform/descriptor/CombinedQuarkusPlatformDescriptor.java

```diff
@@ -79,6 +82,16 @@ public String getQuarkusVersion() {
         return master.getQuarkusVersion();
     }
 
+    @Override
+    public Map<String, Object> getMetadata() {
+        Map<String, Object> metadata = new LinkedHashMap<>();
+        metadata.putAll(master.getMetadata());
+        for (QuarkusPlatformDescriptor platform : platforms) {
+            metadata.putAll(platform.getMetadata());
```

Comment:
1) `master` appears to be `platforms.get(0)`?
2) This does not look consistent with how other pieces (extensions, managed deps) are merged, i.e. in other places what appears first wins while here the last platform overrides the previous one, including the master.

Reply:
Otherwise, it looks good to me.

Reply:
I reversed the put operations, see if that makes sense